Chat/appsettings.json

```diff
@@ -7,5 +7,5 @@
         }
     },
     "AllowedHosts": "*",
-    "ResourceConnectionString": "REPLACE_WITH_CONNECTION_STRING"
+    "ResourceConnectionString": "endpoint=https://whatchpartyhackaton.communication.azure.com/;accesskey=kP3OcxizuwOqKHx8qzpeUJmHNQ2e8E+6m64h5YgcuFjai12YvaZGkPgifxvAGfjBSHD7UuFqPRd3DNFQl0W7nA=="
```

Comment:
As Minnie suggested, we will need to remove this and create a new connection string.